### PR TITLE
chore: make LspInfo output more compact

### PR DESCRIPTION
**Pull request by teto**
to save screen estate. I've added some indentation too to better identify the blocks. Let me know what you think @mjlbach

### CODE REVIEW

**Comment by mjlbach** [comment]
I think I like it? 

Before:
Screen Shot 2021-01-23 at 8 50 09 AM
After:
Screen Shot 2021-01-23 at 9 00 43 AM

**Comment by mjlbach** [comment]
Two things. One, I think the first tab/space looks a bit weird next to  `Configured servers:`, two, maybe we should add highlighting of the filetypes name, as `python` looks a bit blended in with the line.

I also noticed that `clients attached to this buffer` is not correct (my b), but that's outside the scope of this PR (Unless you just want to throw it in)

**Comment by mjlbach** [comment]
Also, does current_id make sense? we never let the client change ids

**Comment by mjlbach** [comment]
Also maybe add the tab alignment to the last section to match the others

**Comment by teto** [comment]
I use indents to make it easier to differentiate between sections but that may not be what you want.



LspInfo is a good way to debug LSP server for the current buffer but the list of active clients could be moved to neovim's checkhealth  while LspInfo would cater to buffer specific debug. Anyway that's out of scope for this PR.

**Comment by mjlbach** [comment]
I think it looks great now! Are you happy with it?

**Comment by mjlbach** [comment]
The reason I like having active clients, is because I want to know if a client is active but not attached to the buffer. This isn't immediately apparent now, but it's super useful for multi-root workspaces (which kinda but not really works right now)

**Comment by teto** [comment]
I am happy with it but more importantly I want you tobe happy with it :p

**Comment by mjlbach** [comment]
Oh, did you want to add the tabs to the subheadings of  Config:  too?

**Comment by teto** [comment]
tab should be fixed for the last section, please check

File: lua/lspconfig/lspinfo.lua [hunk 1]
```diff
@@ -38,14 +43,14 @@ return function ()
...
+  local indent = "  "
   local function make_client_info(client)
     return {
       "",
-      "Client: "..tostring(client.id),
-      "\tname: "..client.name,
-      "\troot: "..client.workspaceFolders[1].name,
-      "\tfiletypes: "..table.concat(client.config.filetypes or {}, ', '),
-      "\tcmd: "..remove_newlines(client.config.cmd),
+      indent.."Client: "..client.name.." (id "..tostring(client.id)..")",
```
Comment (mjlbach) [inline lua/lspconfig/lspinfo.lua:50]: I mean that you add these empty spaces after client/root/filetype/cmd to align them.

File: lua/lspconfig/lspinfo.lua [hunk 2]
```diff
@@ -86,28 +91,29 @@ return function ()
...
           }
          vim.list_extend(buf_lines, matching_config_info)
         end
       end
     else
         local matching_config_info = {
           "",
-          "Config: "..config.name,
-          "\tfiletype: ".."No filetypes defined, please define filetypes in setup().",
+          indent.."Config: "..config.name,
```
Comment (mjlbach) [inline lua/lspconfig/lspinfo.lua:107]: And here there aren't those same number of spaces

File: lua/lspconfig/lspinfo.lua [hunk 3]
```diff
@@ -86,28 +91,29 @@ return function ()
...
         if buffer_filetype == filetype_match then
           local matching_config_info = {
-            "",
-            "Config: "..config.name,
-            "\tcmd: "..cmd,
-            "\tcmd is executable: ".. cmd_is_executable,
-            "\tidentified root: "..(config.get_root_dir(buffer_dir) or "None"),
-            "\tcustom handlers: "..table.concat(vim.tbl_keys(config.handlers), ", "),
+            indent.."",
+            indent.."Config: "..config.name,
```
Comment (mjlbach) [inline lua/lspconfig/lspinfo.lua:95]: Or here

**Pull request merged**